tgui/packages/tgui/interfaces/PaperSheet.js

```diff
@@ -401,29 +397,29 @@ class PaperSheetEdit extends Component {
     } = data;
     const out = { text: text };
     // check if we are adding to paper, if not
-    // we still have to check if somone entered something
+    // we still have to check if someone entered something
     // into the fields
     value = value.trim();
     if (value.length > 0) {
       // First lets make sure it ends in a new line
       value += value[value.length] === "\n" ? " \n" : "\n \n";
```

Comment:
This confuses me. `value[value.length]` shouldn't exist (because JS strings are 0-indexed, so e.g. for length 5 then `value[4]` is the last character). I know you didn't touch this bit, would appreciate you taking a look into if you can as the current code isn't used anywhere (so while you may not be touching this, you're still "introducing" it with this PR).